### PR TITLE
NEW Use template for notification to fixed email

### PR DESCRIPTION
**Pull request by FlorentPoinsaut**
# NEW Use template for notification to fixed email

### CODE REVIEW

File: htdocs/core/lib/notify.lib.php [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+		'',
+		'',
+		$trackid,
+		'',
+		'notification'
+	);
+
+	if ($mailfile->sendfile()) {
+		$sql = "INSERT INTO ".$db->prefix()."notify (daten, fk_action, fk_soc, ".$fkContact."type, objet_type, type_target, objet_id, email)";
+		$sql .= " VALUES ('".$db->idate(dol_now())."', ".$actionId.", ".($object->socid > 0 ? ((int) $object->socid) : 'null').", ".(($contactId) ? $contactId.', ' : '')."'".$db->escape($type)."', '".$db->escape($objectType)."', '".$db->escape($targetType)."', ".((int) $object->id).", '".$db->escape($email)."')";
```
Comment (eldy) [inline htdocs/core/lib/notify.lib.php:264]: use (int) before $actionID and $contactID  (all param in sql must use ->escape or (int) or ->idate

Comment (FlorentPoinsaut) [reply htdocs/core/lib/notify.lib.php]: OK.
Thank you for this feed back!

**Comment by FlorentPoinsaut** [comment]
Hi @eldy!
Could you let me know if you need anything to merge this PR? Or if something goes wrong.
It's a bit of a pain to maintain :)
Thank you for your understanding.

**Comment by eldy** [comment]
Can't merge the PR because the code to send notification has been modified into
notify_sendMail
If feature not yet integradted by another PR, can you check if you can move the code you added into this function ?

**Comment by eldy** [comment]
This PR is now too old, so i close it. A lot of code may have been validated since the PR and the PR may be obsolete, or too difficult to merge due to other changes. Best choice is too cancel it to avoid to keep PR with too low chances to be validated.

**Pull request closed without merging**